gamemode/cl_init.lua

```diff
@@ -16,13 +16,8 @@ local function LoadModules()
     for _, folder in SortedPairs(folders, true) do
         if DarkRP.disabledDefaults["modules"][folder] then continue end
 
-        for _, File in SortedPairs(file.Find(root .. folder .. "/sh_*.lua", "LUA"), true) do
-            if File == "sh_interface.lua" then continue end
-            include(root .. folder .. "/" .. File)
-        end
-
-        for _, File in SortedPairs(file.Find(root .. folder .. "/cl_*.lua", "LUA"), true) do
-            if File == "cl_interface.lua" then continue end
+        for _, File in SortedPairs(file.Find(root .. folder .. "/*.lua", "LUA"), true) do
```

Comment:
Load order has changed here so we are now loading client before shared. Does it matter? Who knows - this is just an observation I wanted to note.